modules/renter/files.go

```diff
@@ -203,8 +212,8 @@ func (r *Renter) FileList() []modules.FileInfo {
 		files = append(files, modules.FileInfo{
 			SiaPath:        f.name,
 			Filesize:       f.size,
-			Available:      f.available(),
-			Redundancy:     f.redundancy(),
+			Available:      f.available(r.offlineContracts),
+			Redundancy:     f.redundancy(r.offlineContracts),
```

Comment:
could we do this instead?
```go
f.available(r.hostContractor.IsOffline)
```
```go
func (f *file) available(isOffline func(types.FileContractID) bool) bool {
    for {
        if isOffline(fc.ID) {
            continue
        }
    }
}
```
That would remove the need to store the offline state on disk, right?